build/NuSpecs/Microsoft.WindowsAppSDK.Foundation.nuspec

```diff
@@ -1,12 +1,12 @@
 <?xml version="1.0" encoding="utf-8"?>
 <package xmlns="http://schemas.microsoft.com/packaging/2010/07/nuspec.xsd">
      <metadata minClientVersion="3.3">
-         <id>Microsoft.WindowsAppSDK.Foundation.TransportPackage</id>
+         <id>Microsoft.WindowsAppSDK.Foundation</id>
          <version>1.0</version>
          <title>$id$</title>
          <authors>Microsoft</authors>
          <description>Windows App SDK</description>
-         <license type="file">LICENSE</license>
+         <license type="file">license.txt</license>
```

Comment:
Is this intentional? I don't think we have a license.txt anywhere in Foundation

Reply:
Yea, this is from             $componentLicenseFilePath = "WindowsAppSDKConfig\NuGetLicense\preview\license.txt"

Reply:
Right I see. But it could be overwritten in build/AzurePipelinesTemplates/WindowsAppSDK-PackTransportPackage-Stage.yml right?

Reply:
Well that stage calls BuildAll which is what places it under the base folder.  And then no one else overwrites it I believe especially give we are in our own folder.